src/sass/layout/_nav.scss

```diff
@@ -1,9 +1,9 @@
 nav{
-    grid-column: 2/3;
-    grid-row: 1/2;
+    display: flex;
+    justify-content: center;
+    align-items: center;
 }
 .barra{
-    display: flex;
     list-style: none;
```

Comment:
remenber to always use english convention from now on, there's no places where they write code in spanish

Reply:
done